packages/editor-ui/src/components/OutputPanel.vue

```diff
@@ -298,6 +327,7 @@ const activatePane = () => {
 		:hide-pagination="outputMode === 'logs'"
 		pane-type="output"
 		:data-output-type="outputMode"
+		:callout-message-key="allToolsWereUnusedNotice"
```

Comment:
This should be just message since you are passing the content and calling i18n.baseText here instead of subcomponent. I am not precious where this should happen, just that the prop name should match expected value. 
```suggestion
		:callout-message="allToolsWereUnusedNotice"
```